### ISSUE TITLE
Show people not in groups

### ISSUE THREAD

**Issue by megoth** [solid/contacts-pane#20]
Today you're able to add people to Address Book without adding them to a group. But only people connected to groups are listed.

Would it be possible to list all people (irregardless of their groups) in the beginning, and use groups more as filters? (Perhaps introducing some paging mechanism if it's to ineffective to load all contacts initally.)

**Comment by timbl** [comment]
Good point.      I guess it may depend on the size of the address book? 

By default just load everyone?


Note: of you have this problem in a particular address book, you can go to the tools pane and use the tool "Put all individuals with no group in a new group".  That will allow you to go though them and classify them.  Beware that is untested and I haven't used it for a while.  That's not a solution for normal people.
